Tests/MathLib/TestCVode.cpp

```diff
@@ -116,11 +116,10 @@ TEST(MathLibCVodeTest, ZeoliteAdsorption)
 	config.put("linear_multistep_method", "BDF");
 	auto ode_solver = MathLib::createOdeSolver<2>(config);
 
+	ode_solver->setFunction(f_zeolite, nullptr);
```

Comment:
Why the reordering? Maybe there will be more changes later on. Let's talk about it.